output/src/main/java/com/scottlogic/deg/output/writer/sql/SqlOutputWriterFactoryFactory.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2019 Scott Logic Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.scottlogic.deg.output.writer.sql;
+
+public class SqlOutputWriterFactoryFactory {
+
+    public SqlOutputWriterFactory create(String tableName) {
+        return new SqlOutputWriterFactory(tableName);
```

Comment:
instead of creating a factory, you can use an ```@Named``` guice to inject the tablename directly into the SqlOutputWriter

Reply:
ah, that sounds good.  I'll look into that

Reply:
done. in 2nd commit